rustls/src/crypto/mod.rs

```diff
@@ -54,6 +55,11 @@ pub trait CryptoProvider: Send + Sync + Debug + 'static {
         &self,
         key_der: PrivateKeyDer<'static>,
     ) -> Result<Arc<dyn SigningKey>, Error>;
+
+    /// Return the signature verification algorithms for use with webpki.
+    ///
+    /// These are used for certificate chain verification, and handshake signature verification.
+    fn certificate_signature_verification_algorithms(&self) -> WebPkiSupportedAlgorithms;
```

Comment:
That's one verbose-ass method name... How do you feel about ditching the `certificate_` prefix?